scripts/xapi-init

```diff
@@ -129,7 +134,7 @@ case "$1" in
 	rhstatus
 	;;
   condrestart)
-	[ -f /var/lock/subsys/xapi ] && restart || :
+	[ -f /var/lock/subsys/xapi ] && restart
```

Comment:
Doesn't the removal ` || :` mean that now the command can fail here, while previously it never did?

Reply:
yes,  just wondering why we always keep it `true`

Reply:
Some of these case can probably be dropped altogether. I think xapi.service only uses `start` and `stop`.

Reply:
yes, I can drop others:
```
[Service]
LimitNOFILE=16384
User=root
Environment="LD_PRELOAD=/usr/lib64/libjemalloc.so.2"
Environment="MALLOC_CONF=narenas:1,tcache:true"
Type=simple
Restart=on-failure
ExecStart=/opt/xensource/libexec/xapi-init start
ExecStartPost=/usr/bin/touch /var/lock/subsys/xapi
#ExecStartPost=/usr/bin/rm -f /etc/xensource/boot_time_info_updated
ExecStop=/opt/xensource/libexec/xapi-init stop
ExecStopPost=/usr/bin/rm -f /var/lock/subsys/xapi /var/run/xapi.pid /var/run/xapi_startup.cookie /var/run/xapi_init_complete.cookie
StandardError=journal
```